src/test/java/uk/gov/ons/ctp/integration/eqlaunch/service/impl/TestEqLaunchService_payloadCreation.java

```diff
@@ -485,4 +496,17 @@ public void createEqLaunchPayload() throws Exception {
     payloadMap.put("exp", "12345");
     return payloadMap;
   }
+
+  public String getEncryptResponseId(String questionnaireId) {
```

Comment:
Since this just repeats the implementation, this method has little value. Replace with a known string result.

Reply:
Removed and used the encrypted code .